core/src/main/java/org/mvcspec/ozark/binding/validate/ConstraintViolations.java

```diff
@@ -118,4 +126,85 @@ else if (lastNode.getKind() == ElementKind.PARAMETER && nodes.size() == 2) {
 
     }
 
+    private static Optional<Annotation[]> setterAnnotationsForField(Class<?> leafBeanClass, String fieldName) {
+        String setterName = setMethodName(fieldName);
+        List<Method> possibleSetters = Arrays.stream(leafBeanClass.getDeclaredMethods())
+            .filter(m -> m.getName().equals(setterName))
+            .collect(Collectors.toList());
+
+        if(possibleSetters.isEmpty())
```

Comment:
Could you please always use `{}` blocks even if the `if` only contains one statement?